Source/Scene/PrimitivePipeline.js

```diff
@@ -272,17 +272,16 @@ define([
         var attributeLocations;
         var instances = parameters.instances;
         var length = instances.length;
+        var pickOffsets;
 
         if (length > 0) {
             geometries = geometryPipeline(parameters);
             if (geometries.length > 0) {
                 attributeLocations = GeometryPipeline.createAttributeLocations(geometries[0]);
             }
-        }
-
-        var pickOffsets;
-        if (parameters.createPickOffsets && geometries.length > 0) {
-            pickOffsets = createInstancePickOffsets(instances, geometries);
+            if (parameters.createPickOffsets && geometries.length > 0) {
```

Comment:
You can get rid of `&& geometries.length > 0` and move this into the above if check.

Reply:
Ah yeah, thanks